packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -141,15 +145,22 @@ export class TestObjectProvider {
      */
     public createLoader(
         packageEntries: Iterable<[IFluidCodeDetails, fluidEntryPoint]>,
-        options?: ILoaderOptions,
+        options?: ITestLoaderOptions,
         detachedBlobStorage?: IDetachedBlobStorage,
     ) {
+        const multiSinkLogger = new MultiSinkLogger();
+        if(options?.logger !== undefined) {
+            multiSinkLogger.addLogger(options.logger);
+            multiSinkLogger.addLogger(ChildLogger.create(getTestLogger?.(),
```

Comment:
i would just always add the child logger, and then always specific the multiSinkLogger  below. only add optiion.logger if no undefined